src/integrationTest/java/uk/gov/hmcts/darts/usermanagement/controller/SecurityGroupControllerIntTest.java

```diff
@@ -105,6 +98,15 @@ void getSecurityGroupsFilterByRoleIdsShouldSucceedAndReturnFilteredGroups() thro
 
         String expectedJson = """
                 [
+                    {
+                     "user_ids":[],
+                     "id":-7,
```

Comment:
Author note: 
this happens because I've introduced a new predefined test security group "Test Judge Global" (with id -7)